.github/workflows/containers.yml

```diff
@@ -0,0 +1,38 @@
+name: Containers
+on:
+  # Be able to run the job manually when needed
+  # (Actions -> Containers -> Run workflow)
+  workflow_dispatch:
+  # Build every week on Monday 00:00
+  schedule:
+    - cron:  '0 0 * * 1'
+env:
+  IMAGE_REGISTRY: quay.io
+  can_push: ${{ github.repository_owner == 'oVirt' }}
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container: [ centos-8 centos-9 fedora-35 fedora-36 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install podman
+      - name: Build container images
+        working-directory: automation/containers
+        run: make ${{ matrix.container }}
+      - name: Push to Quay.io
+        id: push-to-quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ovirt/${{ matrix.container }}
+          tags: ${{ matrix.container }}
```

Comment:
I don't think we have a tag, are you sure this is correct?

Reply:
Yeah, only tags for imageio are the default repository ones. If we don't use tags I can remove this part.